anchor-reference-server/src/main/java/org/stellar/anchor/reference/service/CustomerService.java

```diff
@@ -33,6 +33,14 @@ public GetCustomerResponse getCustomer(GetCustomerRequest request) throws NotFou
       if (maybeCustomer.isEmpty()) {
         throw new NotFoundException(
             String.format("customer for 'id' '%s' not found", request.getId()));
+      } else if (request.getType() != null) {
+        if (!request.getType().equals("sep31-sender")
+            && !request.getType().equals("sep31-receiver")) {
```

Comment:
Not sure if we should be hard-coding those types, this will probably break anchos using different types

Reply:
This is a change in reference (example) implementation. In this example, Business server only accept this 2 types (and reject others)

Reply:
ah this is the reference server, my bad